__tests__/app.test.js

```diff
@@ -159,3 +159,59 @@ describe("GET /api/reviews/:review_id/comments", () => {
       });
   });
 });
+
+describe.only("POST /api/reviews/:review_id/comments", () => {
+  it("201: responds with posted comment object", () => {
+    const commentToPost = {
+      username: "mallionaire",
+      body: "Test Comment For Testing Purposes",
+    };
+    return request(app)
+      .post("/api/reviews/1/comments")
+      .send(commentToPost)
+      .expect(201)
+      .then(({ body }) => {
+        const { comment } = body;
+        expect(comment).toEqual({
+          comment_id: 7,
+          body: "Test Comment For Testing Purposes",
+          review_id: 1,
+          author: "mallionaire",
+          votes: 0,
+          created_at: expect.any(String),
+        });
+      });
+  });
+  it("400: Responds bad request when given a malformed post object (missing necessary values)", () => {
+    const commentToPost = {};
+    return request(app)
+      .post("/api/reviews/1/comments")
+      .send(commentToPost)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe(
+          "400 Bad Request. null value violates not-null constraint"
+        );
+      });
+  });
+  it("400: Responds bad request when trying to post to an invalid review_id (wrong data type) ", () => {
+    const commentToPost = {};
```

Comment:
If you are sad path testing, actually give a comment to post rather than an empty object if that isn't the behaviour you are testing. That way you can be sure that the problem is coming from your intended URL error and not a side effect of posting an empty object! 

I'd do the same for the other test below!